### DISCUSSION TITLE
The emba gets different unzipped files using different profiles

### DISCUSSION THREAD

**Discussion by noxing** [e-m-b-a/emba#620] [Q&A] [answered]
Try to scan the same file, which is actually a bin file. When using full-scan.emba as the configuration file, there are many additional executables in the directory structure and when using default-scan.emba as the configuration file. Why is that? Specifically, there are many more init files in the root directory and many more executable files such as curl in the bin directory

![image](https://github.com/e-m-b-a/emba/assets/50945729/4f080122-2f2a-4e8c-bb99-e5a545ba6ff0)
![image](https://github.com/e-m-b-a/emba/assets/50945729/f266bb12-6059-4218-a600-0fe02168c143)

**Comment by m-1-k-3** [comment]
Can you please share the testing firmware and both EMBA startup commands?

**Comment by noxing** [comment]
It should be like this:

./emba -P 10 -l /log_test -f /xxx.bin -p ./scan-profiles/disable-moudle.emba
./emba -P 10 -l /log_test_1 -f /xxx.bin -p ./scan-profiles/full-scan.emba

**Comment by noxing** [comment]
Will these parameters generate additional files on subsequent scans? Or is there a setup for a simulation run?
export QEMULATION=1
export FULL_EMULATION=1

**Comment by noxing** [comment, marked as answer]
I'm really sorry, because the two filenames are very similar, I made the wrong file, so the above problem occurred
